code/src/components/Background.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+import { useSelector } from 'react-redux';
+// import { loading } from '../reducers/loading'
+
+// const backgroundurls = [tim-roosjen-IBXb__pD17M-unsplash.png, x]
+// background-image: url("../assets/images/tim-roosjen-IBXb-p-D17-M-unsplash.jpg");
+// fixed bg = url("https://i.postimg.cc/2SjZZWrf/tim-roosjen-IBXb-p-D17-M-unsplash.jpg")
+
+const StyledBackground = styled.div`
+    width: 100vw;
+    height: 100vh;
+    background-image: url(${(props) => props.imageUrl});
+    background-position: center; /* Center the image */
+    background-repeat: no-repeat; /* Do not repeat the image */
+    background-size: cover; /* Resize the background image to cover the entire container */
+    position: absolute;
+    top: 0;
+    left: 0;
+    z-index: 1;
+    opacity: 0.8;
+`
+
+const StyledNoBackground = styled.div`
+    width: 100vw;
+    height: 100vh;
+    background-color: #000;
+    position: absolute;
+    top: 0;
+    left: 0;
+    z-index: 1;
+    opacity: 0.8;
+`
+
+const Background = () => {
+  const coordinates = useSelector((store) => store.labyrinth.coordinates)
+  const loading = useSelector((store) => store.loading.isLoading)
+  console.log('coordinates:', coordinates)
+  console.log('loading:', loading)
+
+  let imageUrl;
+
+  if (coordinates === '0,0') {
```

Comment:
I loved how you implemented different backgrounds depending on the coordinates! Nice job!